app/controllers/api/v1/rooms_controller.rb

```diff
@@ -26,6 +26,31 @@ def show
         }, status: :ok
       end
 
+      # POST /api/v1/room/:friendly_id/start.json
+      # Returns: { data: Array[serializable objects] , errors: Array[String] }
+      # Does: Starts the Room session and joins in the room starter.
+
+      def start
+        # TODO: amir - Check the legitimately of the action.
+        bbb_api = BigBlueButtonApi.new
+        room_id = @room.friendly_id
+        session_starter = current_user ? "user(id):#{current_user.id}" : 'unauthenticated user'
+        options = { logoutURL: request.headers['Referer'] || root_url }
+        retries = 0
+        begin
+          logger.info "Starting session for room(friendly_id):#{room_id} by #{session_starter}."
+          join_url = bbb_api.start_session room: @room, session_starter: current_user, options: options
```

Comment:
I would just do it in one line 
```
BigBlueButtonApi.new.start_session room: ....
```

Reply:
That means creating new instance for each retry (including creating new `BigBlueButton::BigBlueButtonApi instance`) plus it further increases the line length and decreases readability.
Do you like me to change it?

Reply:
No that's a fair point. Lets leave it as is